kubos/test/test_build.py

```diff
@@ -0,0 +1,47 @@
+# Kubos CLI
+# Copyright (C) 2016 Kubos Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import kubos
+import mock
+import sys
+import unittest
+import yotta.build
+
+from kubos.test.utils import  KubosTestCase
+
+class KubosBuildTest(KubosTestCase):
+    '''
+    Since the kubos build command proxies to the default yotta implementation
+    this test only looks to make sure that the yotta implementation is called.
+    There's a separate yotta unit test to test the build functionality.
+    '''
+    def setUp(self):
+        super(KubosBuildTest, self).setUp()
+        self.test_function = mock.MagicMock()
```

Comment:
I tinkered with this a little bit and found some interesting behavior when trying to mock both the `KubosTestCase.test_function` and `KubosBuildTest.test_function` attributes with a decorator. 

Rather than investing more time and being blocked on this now I think this should get merged and I can circle back and clean this up on the next unit test iteration at a later date.